receiver/postgresqlreceiver/client.go

```diff
@@ -66,13 +66,62 @@ type client interface {
 	getVersion(ctx context.Context) (string, error)
 	getQuerySamples(ctx context.Context, limit int64, logger *zap.Logger) ([]map[string]any, error)
 	getTopQuery(ctx context.Context, limit int64, logger *zap.Logger) ([]map[string]any, error)
+	explainQuery(query string, queryID string, logger *zap.Logger) (string, error)
 }
 
 type postgreSQLClient struct {
 	client  *sql.DB
 	closeFn func() error
 }
 
+// explainQuery implements client.
+func (c *postgreSQLClient) explainQuery(query string, queryID string, logger *zap.Logger) (string, error) {
+	_, err := c.client.Exec("/* otel-collector-ignore */ SET plan_cache_mode = force_generic_plan;")
```

Comment:
Is this safe to users? Should we add notifications in the doc?

Reply:
yes, this only impact the current session